brightray/browser/win/windows_toast_notification.cc

```diff
@@ -58,12 +58,19 @@ bool WindowsToastNotification::Initialize() {
                                                        &toast_manager_)))
     return false;
 
-  ScopedHString app_id;
-  if (!GetAppUserModelID(&app_id))
-    return false;
+  if (IsRunningInDesktopBridge()) {
+    // Ironically, the Desktop Bridge / UWP environment
+    // does not require us to give Windows an appUserModelId.
```

Comment:
WIth this PR, when running in Desktop Bridge, we don't even try to get the app_id. Shouldn't it be that we always try, but only return `false` if we're not running in desktop mode?

Not requiring this change, only suggesting/asking because it's not clear to me from the comments specifically "does not require" is not the same as "does not accept"

Reply:
We will still have an `app_id` when running in the desktop bridge! It will be automatically assigned by the UWP environment, but we'll still have one.

However, in the UWP environment, creating a ToastNotifier with an appId is explicitly not supported. One _must_ create one without an AUMID passed in, regardless of whether it's correct or not.

Reply:
How about rephrasing to "will not work if we give Windows an appUserModelId".  I believe that's what @felixrieseberg is saying here